lib/msal-node/src/request/ManagedIdentityRequest.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { BaseAuthRequest } from "@azure/msal-common/request/BaseAuthRequest";
+
+/**
+ * ManagedIdentityRequest
+ * - forceRefresh - forces managed identity requests to skip the cache and make network calls if true
+ * - resourceUri  - the URI of the managed identity
```

Comment:
See https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/src/client/Microsoft.Identity.Client/IManagedIdentityApplication.cs 

Please add a task to the board to make sure we support both formats. Can be addressed separately. For now just focus on "resourceUri"

Reply:
I don't understand? forceRefresh is optional here.

Reply:
I am not talking about force refresh, I am talking about the format of the resource URI string (btw, please rename the param as `resource` since it's not a URI ). 

See https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/src/client/Microsoft.Identity.Client/IManagedIdentityApplication.cs#L22